src/Data/HaskellModule.hs

```diff
@@ -32,7 +32,7 @@ import Data.HaskellModule.Parse
 showModule :: FilePath -- ^ the original's filename,
                        --   used for fixing up line numbers
            -> HaskellModule -> B.ByteString
-showModule orig (HaskellModule {..}) = showSource orig fullSource
+showModule orig HaskellModule {..} = showSource orig fullSource
```

Comment:
What is this magic? Parentheses are not required when record wildcards are used?